src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -1316,7 +1322,8 @@ const formConfig = {
           depends: (formData, index) => {
             if (index === undefined) return true;
             return (
-              get('applicantHasOhi', formData?.applicants?.[index]) === 'yes'
+              get('applicantHasOhi.hasOhi', formData?.applicants?.[index]) ===
+              'yes'
```

Comment:
`depends` function needs to drill down into the `hasOhi` property. This was overlooked with the original implementation of this PR, fixed here.